### ISSUE TITLE
Stop trying to figure out the Python environment with global pythonn kernelspecs

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#12278]
With global python kernlespecs, the argv simply has `python` as the path to the executable
We have no idea what python environment should be used.
Instead we 
* Use the active interpreter for global kernelspecs
I.e. if you have a global kernelspec: 
* Open folder a, an you have a .ven in that folder (& thats the active interpreter), then then run a notebook against the global kernlespec then we use .venv]
* Open folder b and have a .conda as the local active python interpreter, now if you run the a notebook against that same notebook, then we use .conda
* Now if you you use the termina and run a notebook against that kernel, then Jupyter will use yet another python environment]

Basically we have no idea what to use, and we try to guess these and the logic is flawed (as we assume the active python env is the right one)

**Suggestion:**
* List python kernels in the kernelspec section of the quick pick
* When users pick these inteepreters we can warn them about the fact that we have no idea what python env is to be used
From there we can either
* Ask the user to update the kerrnel.json file (or we can do that)
* Or other

**Why the change**
* The current code added to guess the interpreter complicates the code

**Comment by DonJayamanne** [comment]
<details>
  <summary>Notes</summary>
  

* Why do we not display default Kernelspecs in Python envs
    * Assume you have a virtual env, the default kernelspec in that env is `python3`
    * We will not display that kernelspec in the list of kernels
    * Why?
    * Assume you have 3 global python envs and each as a deafult kernelspec, now we will end up displaying 4 kernelspecs in the kernels list (one for each python env).
    * This ends up being a duplicate of the python env list, and serves no benefit to the user.
    * To this date, users have never reported that these kernels are not displayed/not accessbile, etc, hence safe to assume that we're doing the right thing

</details>
